libs/kotaemon/kotaemon/loaders/mp3_loader.py

```diff
@@ -0,0 +1,71 @@
+import torch
+from loguru import logger
+from typing import Optional, List
+from kotaemon.base import Document, BaseReader
+from transformers import AutoModelForSpeechSeq2Seq, AutoProcessor, pipeline

```

Comment:
`transformers` and `torch` are not a required package in pyproject.toml at the moment.
Please wrap the import in a try-except and only install it on demand, following this practice in the example below. Else, it will lead to `ModuleNotFoundError`.
https://github.com/Cinnamon/kotaemon/blob/e2bd78e9c44ad86acbf06ea8c8b243e22fb75cbe/libs/kotaemon/kotaemon/loaders/pdf_loader.py#L29-L32

Reply:
I did this changes changes